_pages/saml.md

```diff
@@ -499,6 +501,77 @@ An example decoded logout response, with indentation added for readability.
 </div>
 </div>
 
+### Remote Logout request
+
+Login.gov also offers a remote / back-channel logout endpoint if your
+application needs to log users out without redirecting them back to Login.gov.
+This is still **not** true Single Logout (SLO), it will only terminate a given
+user's session with Login.gov.
```

Comment:
maybe add "you still need to manage the session for your own application" or something?

Reply:
Great point, I'll add it in.